docs/features/generators.md

```diff
@@ -24,8 +25,14 @@ Source generators are implementations of `Microsoft.CodeAnalysis.SourceGenerator
     }
 ```
 `SourceGenerator` implementations are defined in external assemblies passed to the compiler
-using the same `-analyzer:` option used for diagnostic analyzers. An assembly can
-contain a mix of diagnostic analyzers and source generators.
+using the same `-analyzer:` option used for diagnostic analyzers. Valid source generators
+must:
+
+1. Implement `Microsoft.CodeAnalysis.SourceGenerator`
+1. Be decorated with the `Microsoft.CodeAnalysis.SourceGeneratorAttribute` to indicate 
+   supported languages.
```

Comment:
Thanks. Perhaps also add a note about `-feature`, something like: Source generators are currently an experimental feature in the compiler. To enable, include `-feature:replace` on the command line or ```<Features>replace</Features>``` in the `.proj`.